responses.py

```diff
@@ -341,12 +346,14 @@ class RequestsMock(object):
     def __init__(self,
                  assert_all_requests_are_fired=True,
                  response_callback=None,
-                 passthru_prefixes=()):
+                 passthru_prefixes=(),
+                 target='requests.adapters.HTTPAdapter.send'):
```

Comment:
might make sense to just specify the base library for the target? (e.g. requests vs urllib3.vendor.requests)

Reply:
Hmm, I considered that, but couldn't think of any advantage over giving the full control to the client.  For example, somebody may have forked requests and/or subclassed HTTPAdapter, for whatever reason, and there isn't reason to prevent those users from customizing their target correctly.

Would you like me to change it?